### DISCUSSION TITLE
Postgres Schema

### DISCUSSION THREAD

**Discussion by jmarbutt** [prisma/prisma#11122] [Q&A] [answered]
I know this is probably been asked but it is really difficult to search for considering the term `schema` is used a ton in the Prisma docs and issues. But is it possible to use multiple PostgresSQL schemas with Prisma? We have our database organized into multiple schemas to keep things organized because we have hundreds of tables in each. We have them in schemas that represent a specific module in our application. 

Is there a way to do this with Prisma?

**Comment by janpio** [comment, marked as answer]
Not possible yet. Feature is tracked at https://github.com/prisma/prisma/issues/1122

**Comment by jmarbutt** [comment]
Thanks @janpio that will prevent us from using Prisma for now. It is pretty critical in most larger apps to have this separation.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
